packages/contracts/contracts/payoutStrategy/MerklePayoutStrategy.sol

```diff
@@ -0,0 +1,59 @@
+// SPDX-License-Identifier: AGPL-3.0-only
+pragma solidity ^0.8.0;
+
+import "../payoutStrategy/IPayoutStrategy.sol";
+
+import "../utils/MetaPtr.sol";
+
+/**
+ * @notice Merkle Payout Strategy contract which is deployed once per round
+ * and is used to upload the final match distribution.
+ * 
+ * @dev
+ *  - TODO: add function distribute() to actually distribute the funds  
+ */
+contract MerklePayoutStrategy is IPayoutStrategy {
+
+  // --- Data ---
+
+  /// @notice Unix timestamp from when round can accept applications
+  bytes32 public merkleRoot;
+
+  /// @notice Unix timestamp from when round stops accepting applications
+  MetaPtr public distributionMetaPtr;
+
+
+  // --- Event ---
+
+  /// @notice Emitted when the distribution is updated
+  event DistributionUpdated(bytes32 merkleRoot, MetaPtr distributionMetaPtr);
+
+
+  // --- Core methods ---
+
+  /**
+   * @notice Invoked by RoundImplementation to upload distribution to the
+   * payout strategy
+   *
+   * @dev
+   * - should be invoked by RoundImplementation contract
+   * - ideally IPayoutStrategy implementation should emit events after
+   *   distribution is updated
+   * - would be invoked at the end of the roune
+   *
+   * @param _encodedDistribution encoded distribution
+   */
+  function updateDistribution(bytes calldata _encodedDistribution) external override isRoundContract {
```

Comment:
slither is complaining here `Parameter MerklePayoutStrategy.updateDistribution(bytes)._encodedDistribution (contracts/payoutStrategy/MerklePayoutStrategy.sol#46) is not in mixedCase`